packages/framework/next/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "main": "../dist/cjs/servers/next.cjs",
+  "module": "../dist/esm/servers/next.ts",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Needs to use the `.js` extension for `next` module export‏
```suggestion
  "module": "../dist/esm/servers/next.js",
```